src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -100,10 +100,9 @@ function getAjaxStudentList(displayIcon) {
             cache: false,
             data: {
                 courseid: $spreadsheetForm.children(`input[name="${ParamsNames.COURSE_ID}"]`).val(),
-                user: $spreadsheetForm.children(`input[name="${ParamsNames.USER_ID}"]`).val(),
             },
             beforeSend() {
-                displayIcon.html('<img height="25" width="25" src="/images/ajax-preload.gif">');
+                displayStatus.html('<img height="25" width="25" src="/images/ajax-preload.gif">');
```

Comment:
Now that I see this. You should avoid using HTML in Javascript. Just predefine the element in JSP, but hide it by default and show it when needed.

Reply:
Similar comment to https://github.com/TEAMMATES/teammates/pull/8884#discussion_r205968463, other files that have this behavior are i.e `instructorCourses.js`